examples/qualcomm/scripts/ptq_mobilebert_fine_tune.py

```diff
@@ -0,0 +1,314 @@
+# Copyright (c) Qualcomm Innovation Center, Inc.
+# All rights reserved
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import os
+
+import numpy as np
+
+import torch
+from executorch.examples.qualcomm.scripts.utils import (
+    build_executorch_binary,
+    make_output_dir,
+    SimpleADB,
+)
+from transformers import BertTokenizer, MobileBertForSequenceClassification
+
+
+def evaluate(model, data_val):
+    predictions, true_vals = [], []
+    for data in data_val:
+        inputs = {
+            "input_ids": data[0],
+            "attention_mask": data[1],
+            "labels": data[2],
+        }
+        logits = model(**inputs)[1].detach().numpy()
+        label_ids = inputs["labels"].numpy()
+        predictions.append(logits)
+        true_vals.append(label_ids)
+
+    return (
+        np.concatenate(predictions, axis=0),
+        np.concatenate(true_vals, axis=0),
+    )
+
+
+def accuracy_per_class(preds, goldens, labels):
+    labels_inverse = {v: k for k, v in labels.items()}
+    preds_flat = np.argmax(preds, axis=1).flatten()
+    goldens_flat = goldens.flatten()
+
+    for golden in np.unique(goldens_flat):
+        pred = preds_flat[goldens_flat == golden]
+        true = goldens_flat[goldens_flat == golden]
+        print(f"{labels_inverse[golden]}: " f"{len(pred[pred == golden])}/{len(true)}")
+
+
+def get_dataset(data_val):
+    # prepare input data
+    inputs, input_list = [], ""
+    for index, data in enumerate(data_val):
+        inputs.append(tuple(data[:2]))
+        input_text = " ".join(
+            [f"input_{index}_{i}.raw" for i in range(len(inputs[-1]))]
+        )
+        input_list += f"{input_text}\n"
+
+    return inputs, input_list
+
+
+def get_fine_tuned_mobilebert(artifacts_dir):
+    from io import BytesIO
+
+    import pandas as pd
+    import requests
+    from sklearn.model_selection import train_test_split
```

Comment:
do you need sklearn dep? If so, has it been listed in requirements for qnn delegate

Reply:
The mobilebert multi-class text classification example requires pandas and sklearn. We have a mention here.
https://github.com/pytorch/executorch/blob/main/examples/qualcomm/README.md#additional-dependency